cli/tests/unit/performance_test.ts

```diff
@@ -27,6 +27,15 @@ Deno.test(function timeOrigin() {
   assert(Date.now() >= origin);
 });
 
+Deno.test(function performanceToJSON() {
+  const json = performance.toJSON();
+
+  assert("timeOrigin" in json);
+  assert(typeof json.timeOrigin === "number");
```

Comment:
Sorry for not saying this in my original comment: couldn't this be `assert(json.timeOrigin === performance.timeOrigin);`